content/post/graceful-k8s-delpoyments/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Zero-Downtime Rolling Deployments in Kubernetes"
+date: 2021-07-1500:00:00+09:00
+hero: /images/rolling-waves.jpg
+excerpt: |
+  Kubernetes enables us to operate robust systems at scale. However, it is not perfect and in edge cases k8s may behave in unexpected ways.
+authors:
+  - Manish Yadav
+  - Oscar Garcia
+  - Nemo
+  - Boaz Yaniv
```

Comment:
Wow. Is there a primary author?

Reply:
well, it's synthesized from several posts & discussions from discourse so I can't say who is the "main" author (maybe the only thing I can say with confidence is that I'm the editor of it^^).

Reply:
It's like Homer from the Iliad and Odyssey.
We're all Homer 🤣